app/experimenter/nimbus-ui/src/components/FormExperimentOverviewPartial/index.tsx

```diff
@@ -117,28 +151,53 @@ const FormExperimentOverviewPartial = ({
         <FormErrors name="application" />
       </Form.Group>
 
+      {experiment && (
+        <Form.Group controlId="publicDescription">
+          <Form.Label>Public Description</Form.Label>
+          <Form.Control
+            {...nameValidated("publicDescription")}
+            as="textarea"
+            rows={3}
+            defaultValue={experiment?.publicDescription || ""}
+          />
+          <Form.Text className="text-muted">
+            This description will be public to users on about:studies
+          </Form.Text>
+          <FormErrors name="publicDescription" />
+        </Form.Group>
+      )}
+
       <div className="d-flex flex-row-reverse bd-highlight">
+        {onNext && (
```

Comment:
Do we want to display the Next button and the overview submit button on the Overview page outside of `OverviewPartial`? We're going to need to add one additional field post-parity (public description) and if we add that in this file, it's no longer a `partial` but instead is just a `FormOverview` displayed on the "overview" page and the "New" page with conditionals ("overview" shows a new field with a next/submit button, and the "new" page hides that field and instead shows a "create" button). 

If that makes things easier to not deal with nested components/callbacks with this being a "partial" of a form, maybe we should rename this file. Prior to yesterday the "public name" was also going to be hidden on the overview page so the separation between new/overview was slightly more significant.

Reply:
Actually, after staring at this a bit longer I see the GQL call for creating an experiment does indeed live in `PageNew` and the query for `PageEditOverview` can live there so yeah this seems fine - but still wondering if the name of the component makes sense. (and I'm the one that originally suggested the name hehe)